core/store/models/run.go

```diff
@@ -120,17 +120,11 @@ func (jr *JobRun) SetError(err error) {
 func (jr *JobRun) ApplyResult(result RunResult) {
 	jr.Result = result
 	jr.Status = result.Status
-	if jr.Status.Completed() {
-		jr.CompletedAt = null.Time{Time: time.Now(), Valid: true}
-	}
 }
 
-// MarkCompleted sets the JobRun's status to completed and records the
-// completed at time.
-func (jr *JobRun) MarkCompleted() {
-	jr.Status = RunStatusCompleted
-	jr.Result.Status = RunStatusCompleted
-	jr.CompletedAt = null.Time{Time: time.Now(), Valid: true}
+// SetFinishedAt sets the JobRun's finished at time to now.
+func (jr *JobRun) SetFinishedAt() {
+	jr.FinishedAt = null.Time{Time: time.Now(), Valid: true}
```

Comment:
`TimeFrom`

Reply:
Sorry, can you elaborate?

Reply:
`jf.RinishedAt = `null.TimeFrom(time.Now())`